requirements.txt

```diff
@@ -0,0 +1 @@
+pycups==1.9.74
```

Comment:
why fix static version and not a minimal version with >=

Reply:
I upgraded it from the last version (1.9.73 in v12)
It's tested and working with this version, not sure about future versions